addons/web/static/src/core/emoji_picker/emoji_picker.xml

```diff
@@ -82,7 +82,7 @@
 </t>
 
 <t t-name="web.EmojiPicker.searchInput">
-    <input class="form-control border-0 flex-grow-1 rounded-3 rounded-end-0 o-active" placeholder="Search for an emoji" t-model="localState.searchTerm" t-ref="autofocus" t-att-model="localState.searchTerm" t-on-input="() => this.state.activeEmojiIndex = 0"/>
+    <input class="form-control border-0 flex-grow-1 rounded-3 rounded-end-0 o-active" placeholder="Search for an emoji" t-model="localState.searchTerm" t-ref="autofocus" t-att-model="localState.searchTerm" t-on-input="() => this.state.activeEmojiIndex = 0" t-att-tabindex="isMobileOS ? -1 : 0"/>
```

Comment:
`t-ref="autofocus"` is there for `useAutofocus()`, which should not focus on mobile OS.

I don't have an android phone so I cannot test. Some colleagues confirm the bug and this PR solves it... But it would be nice to see why the inner-code of `useAutofocus()` doesn't prevent autofocusing of input in mobile when it has some code to do that.

Reply:
This happens because the dialog uses the `useActiveElement` hook, which retrieves all the tabbable elements within the referenced element and traps the focus there. 
(https://github.com/odoo/odoo/blob/507d58836f9dc46231823a37e57c61b37ad46cc8/addons/web/static/src/core/ui/ui_service.js#L74)
Since the input element is included in the `TABABLE_SELECTOR`, it receives focus as well.
(https://github.com/odoo/odoo/blob/507d58836f9dc46231823a37e57c61b37ad46cc8/addons/web/static/src/core/utils/ui.js#L164)